developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_28.rst

```diff
@@ -54,6 +54,11 @@ Removed APIs
 Back-end changes
 ----------------
 
+PHP 8.3
+^^^^^^^
+
+In this release support for PHP 8.3 was added. See release notes from PHP about new deprecations.
```

Comment:
Nitpick: https://docs.nextcloud.com/server/latest/developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_24.html#php-8-1 was a bit more elaborative. Do we want to copy and adapt those instructions?

Reply:
I copied from 26 8.2 :confused:

Reply:
Done